test/integration/serverless/src/logPython.py

```diff
@@ -1,11 +1,16 @@
 # flake8: noqa
 import logging
+import time
 
 logger = logging.getLogger(__name__)
 
 
 def log(event, context):
+    time.sleep(0.25)
     logger.error("XXX LOG 0 XXX")
+    time.sleep(0.25)
     logger.error("XXX LOG 1 XXX")
+    time.sleep(0.25)
     logger.error("XXX LOG 2 XXX")
+    time.sleep(0.25)
     return {"statusCode": 200, "body": "ok"}
```

Comment:
For all log testing handlers, I've added sleeps between each of the logs. Python and Go log tests were having issues with out of order logs, and this solution seemed better than sorting the log test logs so we didn't have to update the existing snapshots (we sort everything but the log test logs)

Reply:
I'm usually weary of using sleeps to solve race conditions. Besides needing to update the existing snapshots, what's the downside of sorting the logs? That seems like a much better solution but I'm also not an expert on this.